buildSrc/src/main/groovy/openhouse.client-codegen-convention.gradle

```diff
@@ -7,7 +7,7 @@ ext {
   jakarta_annotation_version = "1.3.5"
   reactor_version = "3.4.3"
   jodatime_version = "2.9.9"
-  junit_version = "4.13.1"
+  junit_version = "5.11.0"
```

Comment:
shall we move this to top of `build.gradle` ?

Reply:
good catch, it's in the parent file already, will remove this duplicate